karate.js

```diff
@@ -6,11 +6,14 @@ karate.executable = function () {
 	let prefix = karate.config.dir ? '-Dkarate.config.dir=' + karate.config.dir + ' '  : '';
 	return prefix + 'com.intuit.karate:karate-core:' + karate.version + ':all';
 };
-karate.exec = function (args) {
+karate.exec = function (args, jvmArgs) {
```

Comment:
instead of this, would an API like `karate.jvmArgs = '-Foo=Bar';` be better, similar to how `karate.config` and `karate.version` work.

Reply:
Makes sense. I will make the recommended changes!

Thanks for the feedback

Reply:
I finally found some time to incorporate your feedback. 

Please let me know what you think :)